src/herder/TxSetFrame.cpp

```diff
@@ -36,80 +36,154 @@ namespace stellar
 
 namespace
 {
+std::string
+getTxSetPhaseName(TxSetPhase phase)
+{
+    switch (phase)
+    {
+    case TxSetPhase::CLASSIC:
+        return "classic";
+    case TxSetPhase::SOROBAN:
+        return "soroban";
+    default:
+        throw std::runtime_error("Unknown phase");
+    }
+}
+
+bool
+validateNonParallelPhaseXDRStructure(TransactionPhase const& phase)
```

Comment:
I'm not sure how to name that TBH. I guess I could just say 'v0' phase, would that be more readable?

Reply:
v0 is fine here, but my comment was more about the naming throughout the code. Might be nice to replace all mentions of parallel with conflict-free, and also add a comment that there isn't any parallelization yet to make it clear for the readers.

Reply:
Well, it's not quite conflict-free. I'm fine with using something like v0 or maybe 'sequential' for the existing phases, but I'm not sure if there is a better term than 'parallel' for the new phase, because it is designed to be applicable in parallel (unlike the existing phase). I'm not sure we need to obfuscate that fact just because there is no followup work merged yet.

Reply:
I've used 'sequential' for v0 phase. Still not sure if there is a better way to name the parallel phase.